src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -637,7 +631,7 @@ public class LibraryServiceTest {
     mockLibraryService.setResponses(expectedResponses);
 
     String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
-    String indexName = "indexName746962392";
+    String indexName = "default index";
```

Comment:
As discussed earlier, we would prefer not to initialize all fields of the response since the marginal benefit of dong this is small. And it is preferable to keep the test short and efficient while the functionality is similar.